packages/xo-web/src/index.scss

```diff
@@ -213,6 +213,14 @@ $select-input-height: 40px; // Bootstrap input height
   border: 1px solid black;
   margin: 5px 10px;
   width: 250px;
+  // "input-group > form-control" and "input-group > input-group-btn > btn" have hidden the notification.
+  // (https://github.com/vatesfr/xen-orchestra/issues/3874)
+  // In bootstrap, "input-group > form-control" and "input-group > input-group-btn > btn"
+  // have "z-index: 2" and "z-index: 3" if they are focused.
+  // (https://github.com/twbs/bootstrap/blob/v4.0.0-alpha.5/scss/_input-group.scss#L18ToL37)
+  // (https://github.com/twbs/bootstrap/blob/v4.0.0-alpha.5/scss/_input-group.scss#L177TOL187)
```

Comment:
Use `-` as the line numbers separator (`L18-L37`, `L177-L187`). Any separator seems to work but I guess we can/should just use the most common one.